index.html

```diff
@@ -187,26 +187,27 @@ <h2 class="about-grid-item-title">Skills</h2>
         </div>
       </div>
     </div>
-    <div id="contact-div">
-      <div id="contact-center">
-        <div id="invite"><p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p></div>
-        <div id="contact-form-div">
-          <form id = "survey-form">
-            <input id="fullname" class="form name" type="text" placeholder="Full Name" required>
-            <input id="first-name" class="form name desktop" type="text" placeholder="First Name" required><input id="last-name" class="form name desktop" type="text" placeholder="Last Name" required><br>
-            <input id="email" type="email" class="form"  placeholder="Email adress" name="email" required><br>
-            <textarea id="comments" class="form" name="comments" placeholder="Hey I’d like to connect to..." required></textarea><br>
-            <div id="submit-button" class="grid-button-container">
-              <input type="submit" id="submit" class="grid-button" value="Get in touch">
-            </div>
-            <div id="email-para">
-              <p>nyldrift@hotmail.com</p>
-            </div>
-          </form>
+  </section>
+  <section id="contact">
+    <div id="contact-center">
+      <div id="invite"><p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p></div>
+      <div id="contact-form-div">
+        <form id = "survey-form" action="https://formspree.io/f/mzbqjbpr" method="POST">
+          <input name="First Name" id="first-name" class="form name" type="text" placeholder="First Name" maxlength="30" required>
+          <input id="Last Name" class="form name desktop" type="text" placeholder="Last Name" required><br>
+          <input name="Email" id="email" type="email" class="form"  placeholder="Email adress" name="email" required><br>
+          <textarea name="Comments" id="comments" class="form" placeholder="Hey I’d like to connect to..." maxlength="500" required></textarea><br>
+          <div id="submit-button" class="grid-button-container">
+            <input type="submit" id="submit" class="grid-button" value="Get in touch">
+          </div>
+        </form>
+        <div id="email-para">
+          <p>nyldrift@hotmail.com</p>
```

Comment:
- [ ] Please increase the `font-size` and `letter-spacing`of your email in the footer to comply to the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements):

> - You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 

Figma design :

<img width="308" alt="Screenshot 2023-03-29 at 9 09 12 AM" src="https://user-images.githubusercontent.com/44798044/228424752-d5c5e7c7-5c3b-4268-95e0-e270bf2c3c95.png">

Your design:

<img width="421" alt="Screenshot 2023-03-29 at 9 09 54 AM" src="https://user-images.githubusercontent.com/44798044/228424731-2b875480-75b8-4ec9-a601-3dfeaed97308.png">